CMakeLists.txt

```diff
@@ -225,10 +225,10 @@ endif()
 
 find_package(libyuv QUIET) # not required
 if(libyuv_FOUND)
-    # libyuv 1755 exposed all of the I*Matrix() functions, which libavif relies on.
+    # libyuv 1778 exposed ScalePlane_12 function, which libavif relies on.
```

Comment:
libyuv 1778 exposed I420Scale_12 and I444Scale_12, not ScalePlane12:
https://chromium-review.googlesource.com/c/libyuv/libyuv/+/2706966

ScalePlane_12 was added in libyuv 1774:
https://chromium-review.googlesource.com/c/libyuv/libyuv/+/2658097

It is certainly fine to require a newer version such as libyuv 1778. I am just wondering if you know something wrong with the ScalePlane_12() function in libyuv 1774, such as it was an incomplete or buggy implementation.

Reply:
I recommended to use 1789:
There's some issues in the neon64 assembly when building with gcc:
https://chromium-review.googlesource.com/c/libyuv/libyuv/+/2922147

Reply:
@wantehchang You are right, actually I only did quick commit msg search, and Scale 12 matched on https://chromium.googlesource.com/libyuv/libyuv/+/08815a29766a78398a8e2b9ed095280e9d0a73c2^!/. Grepping the commit diff for `ScalePlane_12` was further positive, and this commit allowed libavif to compile, so I thought this was good. But indeed looking more deeply at the diff I now see that `ScalePlane_12` is called but not defined in this commit (which is as you said about `I420/I444Scale_12`), you are evidently right that 1774 should be enough. But I can update the MR to 1789 instead to avoid the issue mentionned by @tongyuantongyu.

Reply:
Apparently they are other neon fixes in 1796: https://chromium-review.googlesource.com/c/libyuv/libyuv/+/3216739